slick/src/main/scala/slick/lifted/TupleOptionMapper.scala

```diff
@@ -0,0 +1,26 @@
+package slick.lifted
+
+import annotation.implicitNotFound
+import slick.ast.OptionApply
+
+
+
+@implicitNotFound("Cannot perform option-mapped operation\n      with type: ${R}\n  for base type: ${BR}")
+trait TupleOptionMapper[BR, R] extends OptionMapper[BR, R]
```

Comment:
I don't really get this what makes `TupleOptionMapper` different from `OptionMapper`?
This file does not really seem to have any code related specifically to tuples, or am I missing something?

Reply:
Maybe this trait just needs some documentation what it is used/needed for

Reply:
I cannot use OptionMapper2 as OptionMapper, because tuple doesn't have related BaseTypedType.       I need some time to think about this .